tests/build_/test_arrangements.py

```diff
@@ -113,7 +130,29 @@ def get_build_request(self, build_type, osbs,  # noqa:F811
 
         params.update(additional_params or {})
         params['arrangement_version'] = self.ARRANGEMENT_VERSION
-        return params, fn(**params).json
+        if self.ARRANGEMENT_VERSION < REACTOR_CONFIG_ARRANGEMENT_VERSION:
```

Comment:
I'm wondering if `TestArrangementV6` should extend this method:

```python
def get_build_request(self, *args, **kwargs):
  params, build_json = super(TestArrangementV6).get_build_request(*args, **kwargs)
  ...
```

Reply:
Fixed.

Reply:
I made a similar adjustment to test_running_order